app/controllers/api/v0/organizations_controller.rb

```diff
@@ -1,16 +1,37 @@
 module Api
   module V0
     class OrganizationsController < ApiController
+      before_action :find_organization, only: %i[users]
+
       SHOW_ATTRIBUTES_FOR_SERIALIZATION = %i[
         username name summary twitter_username github_username url
         location created_at profile_image tech_stack tag_line story
       ].freeze
       private_constant :SHOW_ATTRIBUTES_FOR_SERIALIZATION
 
+      ORG_USERS_FOR_SERIALIZATION = %i[
```

Comment:
I wouldn't make a difference between "regular users" and "organization users". As I don't think we plan to tell the API what's their role (admins, non admins) for privacy reasons, I think we can render the same partial we render for `/api/users/:id` in the `Api::V0::UsersController#show` route in a loop in the jbuilder file

Let me know if that makes sense

Reply:
Makes sense. I just pushed up a fix for this